### ISSUE TITLE
Sorting of facets

### ISSUE THREAD

**Issue by hagenbauer** [YousefED/ElasticUI#47]
Is there a way to have the facets not sorted by number of documents but alphabeticaly?

Regards

Hagen

**Comment by YousefED** [comment]
Hi! Please refer to the Order part of the relevant aggregation, e.g.: https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-terms-aggregation.html

**Comment by EssentialMusic** [comment]
Hi @hagenbauer

Did you have any joy with this? I'm trying to sort my facets alphabetically as well but am having trouble. 

I've tried something like this
```eui-aggregation="ejs.TermsAggregation('country').field('country').size(0).order('country','desc')```
But I receive a 500 server error.

**Comment by hagenbauer** [comment]
@EssentialMusic no, I gave it a try but i was not successfull

**Comment by YousefED** [comment]
If you debug the network request sent and received from ES (Chrome Inspector -> network tab), I might be able to dig in and help

**Comment by EssentialMusic** [comment]
Thanks both for the quick response. I've actually now solved this!

@YousefED I was looking at the network inspector yesterday and got nowhere. Had another look this morning and decided to copy my query into ES-Head to try and debug there and also looking at the ES docs again: https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-terms-aggregation.html#search-aggregations-bucket-terms-aggregation-order

Basically, where I had ```.order('country','desc')``` I should have been using ```.order('_term','desc')```

Hope this also helps you @hagenbauer

**Comment by jviele** [comment]
Hi guys
sorry - can't figure out: 
<body ng-app="tutorial" eui-index="'uzh_zora_3'" eui-enabled="true" eui-aggregation="ejs.TermsAggregation('date_year').field('date_year').size(0).order('_term','desc')>

with date_year= 2005 or 2008 still shows first "2005", after "2008" :-( Any Idea?
Kind Regards

**Comment by jviele** [comment]
curl -k -XGET 'https://somehost/somecontent/_search?&pretty=true&size=3' -d '{
    "aggs" : {
        "year" : {
            "terms" : {
                "field" : "date_year",
                "order" : { "_term" : "desc" }
            }
        }
    }
}'

works, but howto include in ElasticUI?
